test/controllers/due.diligence.controller.spec.ts

```diff
@@ -600,6 +600,8 @@ describe("DUE_DILIGENCE controller", () => {
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(ErrorMessages.NAME_INVALID_CHARACTERS);
       expect(resp.text).toContain(ErrorMessages.EMAIL_INVALID_FORMAT);
+      expect(resp.text).toContain(ErrorMessages.INVALID_AGENT_ASSURANCE_CODE);
```

Comment:
we'll also need to update a test in   overseas.entity.due.diligence.controller.spec.ts to check the invalid chars error for aml number